src/cogs/rules.py

```diff
@@ -29,24 +33,47 @@ async def rule(self, ctx, number):
         else:
             await ctx.send(f"**Rule {number}**:\n{self._rules[number]}")
 
+    def _change_msg(self, state, status):
+        user = str(state[1].author)
+        ts = datetime.fromtimestamp(state[0]).isoformat(" ", "seconds")
+        content = state[1].clean_content
+        return f"**{discord.utils.escape_markdown(discord.utils.escape_mentions(user))}** {status} on {ts} UTC:\n{content}\n"
+
```

Comment:
what is `state`?

Reply:
It's a point in the history of a message

Reply:
Renamed it to `entry` to hopefully be less confusing